plugins/outputs/zabbix/README.md

```diff
@@ -0,0 +1,406 @@
+# Zabbix Output Plugin
+
+This plugin send metrics to [Zabbix](https://www.zabbix.com/) via
+[traps][traps].
+
+It has been tested with versions
+[3.0](https://www.zabbix.com/documentation/3.0/en/manual/appendix/items/trapper)
+,
+[4.0](https://www.zabbix.com/documentation/4.0/en/manual/appendix/items/trapper)
+and
+[6.0](https://www.zabbix.com/documentation/6.0/en/manual/appendix/items/trapper)
+.
+
+[traps]: https://www.zabbix.com/documentation/current/en/manual/appendix/items/trapper
+
+It should work with newer versions as long as Zabbix does not change the
+protocol.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Send metrics to Zabbix
+[[outputs.zabbix]]
+  ## Address and (optional) port of the Zabbix server
+  address = "zabbix.example.com:10051"
+
+  ## Send metrics as type "Zabbix agent (active)"
+  # agent_active = false
+
+  ## Add prefix to all keys sent to Zabbix.
+  # prefix = "telegraf."
+
+  ## Name of the tag that contains the host name
+  # host_tag = "host"
+
+  ## Skip measurement prefix to all keys sent to Zabbix.
+  # skip_measurement_prefix = false
+
+  ## This field will be sent as HostMetadata to Zabbix Server to autoregister the host.
+  ## To enable this feature, this option must be set to a value other than "".
+  # autoregister = ""
```

Comment:
How does a user know what this field should be? It is non-obvious to me, where I would get this data or what to put here. When I hear metadata, I think I possibly have a key/value set of values I can set.

After reading the section below and the first part of the [autoregistration page](https://www.zabbix.com/documentation/current/en/manual/discovery/auto_registration?hl=autoregistration). It seems this field is about providing an extra value for a host?